content/_includes/server-config.md

```diff
@@ -63,12 +63,12 @@ tls:
 # The API configuration. The APIs exposed by the KES server can
 # be adjusted here. Each API is identified by its API path.
 #
-# In general, the KES server uses reasonable defaults for all APIs.
+# In general, the KES server uses sane defaults for all APIs.
```

Comment:
Maybe a copy/paste error from somewhere, I recall we fixed this a while back
```suggestion
# In general, the KES server uses reasonable defaults for all APIs.
```

Reply:
I'll go ahead and change it, but i copy/paste this from the KES repo. It'll need to change over there for long-term effects. Because I will forget to change it here every time.